core/domain/exp_domain.py

```diff
@@ -547,6 +549,11 @@ def validate(self, obj_type, exp_param_specs_dict):
             except StopIteration:
                 raise utils.ValidationError(
                     'Unrecognized rule type: %s' % rule_spec.rule_type)
+            if rule_class.__name__ == rule_domain.FUZZY_RULE_TYPE:
+                if seen_fuzzy_rule:
+                    raise utils.ValidationError(
+                        'AnswerGroup can only have one fuzzy rule.')
```

Comment:
I was going for "the current AnswerGroup", but don't think this is worth a big standards discussion :), so done.